### PR TITLE
Add tenant admin username and role to the Configurations API response

### PR DESCRIPTION
**Pull request by ayshsandu**
## Purpose
Fix: https://github.com/wso2/product-is/issues/10702

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml [hunk 1]
```diff
@@ -393,6 +396,24 @@ components:
...
+            readOnly: true
+        adminRole:
+            type: string
+            description: "Name of the super admin role in the system."
+            example: "Internal/admin"
+            readOnly: true
+        everyoneRole:
+            type: string
+            description: "Name of the everyone role in the system."
+            example: "Internal/everyone"
```
Comment (IsuraD) [inline components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml:415]: is this everyone or Internal/everyone ?

File: components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml [hunk 2]
```diff
@@ -393,6 +396,24 @@ components:
...
+      properties:
+        adminUser:
+            type: string
+            description: "Username of the super admin user in the system."
+            example: "admin"
+            readOnly: true
+        adminRole:
+            type: string
+            description: "Name of the super admin role in the system."
+            example: "Internal/admin"
```
Comment (IsuraD) [inline components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml:410]: is this admin or Internal/admin" ?

**Review by IsuraD — APPROVED** [review]

File: components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml (hunk 2 above)
Comment (ayshsandu) [reply components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml:410]: `Internal/admin`. This is the value returned by the osgi service.

File: components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml (hunk 1 above)
Comment (ayshsandu) [reply components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/resources/configs.yaml:415]: `Internal/everyone`. This is the value returned by the osgi service.

**Pull request merged**